lib/minitest_ext/exercism_plugin.rb

```diff
@@ -1,8 +1,7 @@
 module Minitest
-  def self.plugin_bogus_options(opts, options)
-  end
+  def self.plugin_bogus_options(opts, options); end
 
-  def self.plugin_exercism_init(*args)
+  def self.plugin_exercism_init(*_args)
```

Comment:
Why this change? Is this Rubocop?

Reply:
Yes:

```bash
== lib/minitest_ext/exercism_plugin.rb ==
W:  4: 34: [Corrected] Lint/UnusedMethodArgument: Unused method argument - args. If it's necessary, use _ or _args as an argument name to indicate that it won't be used. You can also write as plugin_exercism_init(*) if you want the method to accept any arguments but don't care about them.
```

Reply:
Historically, I think, prefer `*_` as opposed to `*_args` because context defines that they are "arguments".  And we avoid the poor name of "args".

Reply:
> You can also write as `plugin_exercism_init(*)`

Let's do this. I didn't know it was possible but I like it.

Reply:
The `*` means that you can't use what is given, while the `*_` gives you at least something to work with if you need to, and is idiomatic, though it does communicate "something we are accept but don't use" in block and method argument lists.

Reply:
> while the *_ gives you at least something to work with if you need to

I think at that stage, the method should just be renamed to `*args` though. I prefer enforcing that we *can't* use them, if the intention is not to use them.